k8s/prod/common/ts/identity.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+  name: ts
+  namespace: ts
+spec:
+  type: 0
+  resourceID: "/subscriptions/1c4f0704-a29e-403d-b719-b90c34ef14c9/resourcegroups/managed-identities-aat-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/ts-aat-mi"
+  clientID: "30a7b8f4-547a-4c3a-b839-aea6fa8066cf"
```

Comment:
Values to be updated after infra built - dependency on https://github.com/hmcts/cnp-jenkins-config/pull/778